src/NMEA/EngineState.hpp

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+#include "Validity.hpp"
+#include "time/Stamp.hpp"
+#include "Atmosphere/Temperature.hpp"
+
+/** Information about the engine sensor box values,
+ *  temperatures and revolutions per second.
+ *  Update rate of the engine sensor box is 1 Hz.
```

Comment:
It would be useful to document the update rate of your product somewhere, but this is the wrong place. This should be a generic struct holding information about the engine. It should be possible to use this struct for other engine sensors. Having this information here is too specific for no reason, because what if another guy makes a sensor box with 10 Hz update rate - will they be able to use this struct, or will they have to invent a new one?